tools/convert/helpers.go

```diff
@@ -197,6 +198,11 @@ func nonZero(data map[string]any, key, oldkey string, example string) string {
 	return fmt.Sprintf(`# %s: %v`, key, yamlf(example))
 }
 
+func now() string {
+	t := time.Now()
+	return fmt.Sprintf("on %s at %s", t.Format("2006-01-02"), t.Format("15:04:05 MST"))
```

Comment:
Lets use UTC like https://github.com/open-telemetry/opentelemetry-operator/blob/846046ccccf3d2bde63be9cb7f75537af5332638/bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml#L34

Reply:
OK...I didn't want to use the RFC3339 format because these are supposed to be human-readable. But using UTC is the right thing.